CMake/DownloadDoxygen.cmake

```diff
@@ -1,3 +1,3 @@
-file( DOWNLOAD http://public.kitware.com/pub/itk/NightlyDoxygen/InsightDoxygenDocHtml.tar.gz
+file( DOWNLOAD https://public.kitware.com/pub/itk/NightlyDoxygen/InsightDoxygenDocHtml.tar.gz
```

Comment:
Was this outdated link somehow responsible for "DoS attack" on Kitware website a few days ago?

Reply:
Perhaps?  I don't know for sure.